manifests/init.pp

```diff
@@ -92,12 +92,9 @@
     fail('cfg parameter must be a hash')
   }
 
+  anchor { 'grafana::begin': } ->
```

Comment:
@dhoppe Great work on the modulesync PR.  All looks good.  This did stand out a bit though.

I thought perhaps we preferred `contain` over `anchor`?

ie like this
```puppet
contain '::grafana::install'
contain '::grafana::config'
contain '::grafana::service'

Class['::grafana::install'] ->
Class['::grafana::config'] ~>
Class['::grafana::service']
```